methods_to_test.js

```diff
@@ -0,0 +1,69 @@
+const capitalize = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+const reverseString = (string) => {
+    let splitString = string.split("")
+    let reversedArray = splitString.reverse()
+    let joinString = reversedArray.join("")
+
+    return joinString
+}
+
+const analyse = (numbers) => {
+    let sum = numbers.reduce((previous, current) => current += previous) / numbers.length
+    // let avrg = sum / numbers.length
+    
+    return {
+        average: sum,
+        min: Math.min(...numbers),
+        max: Math.max(...numbers),
+        length: numbers.length
+    }
+}
+
+class Calculator {
+
+    add(...numbers) {
+        return numbers.reduce((prev, cur) => cur *= prev)
```

Comment:
I am extremely sure that this operation isn't "adding" numbers.

I think the test case we wrote for this has played a trick on us. :D